product_docs/docs/efm/4/upgrading.mdx

```diff
@@ -33,6 +33,8 @@ Failover Manager provides a utility to assist you when upgrading a cluster manag
    ```shell
    /usr/efm-4.4/bin/efm stop-cluster efm
    ```
+!!! Note
+    The primary agent will not drop the virtual IP address (if used) when it is stopped.
```

Comment:
What's the rationale for noting this here? Is there a scenario where dropping the VIP during an upgrade is desirable / necessary?

Reply:
It's so they don't think there is db downtime when upgrading efm.

Reply:
Let's be explicit about that then and give the note context:

```suggestion
    The primary agent will not drop the virtual IP address (if used) when it is stopped; the database will remain up and accessible on the VIP during the EFM upgrade. See also: [Using Failover Manager with virtual IP addresses](04_configuring_efm/05_using_vip_addresses.mdx) 
```